src/functions/_hu-format-modules.scss

```diff
@@ -1,4 +1,4 @@
-$hu-format-modules-order: (base, visited, focus, hover, hocus, active, state, group-focus, group-hover, group-hocus, group-state, reduced-motion, print);
+$hu-format-modules-order: (base, visited, focus, hover, hocus, active, state, group-focus, group-hover, group-hocus, group-state, group-visited, reduced-motion, print);
```

Comment:
For non-responsive classes, this is what correctly orders the modules and which will correctly maintain the LVHA system:

```suggestion
$hu-format-modules-order: (base, visited, focus, hover, hocus, active, state, group-visited, group-focus, group-hover, group-hocus, group-state, reduced-motion, print);
```